lib/hammer_cli_katello/host_package_group.rb

```diff
@@ -1,48 +1,22 @@
 module HammerCLIKatello
   class HostPackageGroup < HammerCLIKatello::Command
-    desc "Manage package-groups on your hosts"
+    desc "Manage package-groups on your hosts. Note these command are no longer available"\
+          "\n Use the remote execution equivalent"
 
     class InstallCommand < HammerCLIKatello::SingleResourceCommand
-      include HammerCLIForemanTasks::Async
-      resource :host_packages, :install
       command_name "install"
-      success_message "Package-groups installed successfully."
-      failure_message "Could not install package-groups"
-
-      validate_options do
-        option(:option_groups).required
-      end
-
-      build_options :without => [:packages]
-
       def execute
-        warn "This command uses katello agent and will be removed in favor of remote execution " \
-          "in Katello 4.10."
-        warn "The remote execution equivalent is `hammer job-invocation create --feature " \
+        warn "Use the remote execution equivalent `hammer job-invocation create --feature " \
```

Comment:
Should this also be `fail`?
```suggestion
        fail "Use the remote execution equivalent `hammer job-invocation create --feature " \
```